### ISSUE TITLE
wk4 task GMB graphics

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#1389]
Use LucidChart (or R - based on what will be quickest) to update the below in TeamPSD style:
- [ ] hexagons
- [ ] bar charts
- [ ] table of hypotheses

Waiting on @lzim for confirmation on style. 
See first three graphics here - https://osf.io/2ha6d/

**Comment by staceypark** [comment]
putting this on hold with r21 manuscripts as a priority instead

**Comment by lzim** [comment]
@staceypark GMB is an R21 manuscript & it would be a great one to get out the door.

We can touch base about where this fits in given all the other capacity constraints.
